Packs/FlashpointFeed/README.md

```diff
@@ -0,0 +1,21 @@
+Access Flashpoint's Technical Indicator dataset, which provides users access to indicators of compromise (IOCs) and technical data across Flashpoint datasets and those included in Finished Intelligence Reports, allowing for seamless integration into users' workflows and automated tools.
+
+Flashpoint is the globally trusted leader in risk intelligence for the fastest, most comprehensive coverage of threatening activity on the internet. From bolstering cyber and physical security, to detecting fraud and insider threats, Flashpoint partners with customers across the private and public sectors to help them rapidly identify threats and mitigate their most critical security risks.
+ 
+### What does this pack do?
+
+Flashpoint Feed Integration allows importing indicators of compromise that occur in the context of an event on the Flashpoint platform. 
+
+The following types of indicators are retrieved:
+
+- IP
+- Domain
+- URL
+- Email
+- SHA1
+- SHA256
+- MD5
```

Comment:
```suggestion
- File
```
We currently only produce File indicator type and not SHA1 and SHA256 and MD5
And this is also what the `auto_detect_indicator_type` function you use does.